### ISSUE TITLE
Create the Best Contribution Guidelines Document

### ISSUE THREAD

**Issue by praveenscience** [praveenscience/Internship-LMS-FrontEnd#61]
Hey folks, Welcome to GSSoC.

This is going to be the second issue, a major one again because this impacts a lot of people. Let's try to create the best Contribution Guidelines for our project possible.

Please, tell me what all sections and information we can include in the Contribution Guidelines document before claiming the issue blindly. Blind claim requests not entertained and will be rejected outright! 😅

I really love this [document: **Contributing to Rezume**](https://github.com/CatsInTech/Rezume/blob/master/CONTRIBUTING.md), so shall we please kindly stick to this? I would love to have a macOS or Ubuntu taking screenshots than Windows because of the Clear Type fonts. Sorry Windows.

Please feel free to let me know what all we can add, this will give you more chance for getting this assigned to you. Let's have a fair assignment for this. Also, this is going to be One Person's work, so it will be assigned for only one person. 😁

Up for grabs! ✋🏻

**Comment by aritra120** [comment]
Sir , Personally I use Windows but I have a contribution guideline ready with me that is in a very generalized form and applicable for windows, mac & ubuntu as well. You can check that for once.

**Comment by praveenscience** [comment]
@aritra120 Sure, can you send me the link, please? 😁

**Comment by aritra120** [comment]
https://github.com/firstcontributions/first-contributions
Here is the link. Just go through it for once.

**Comment by praveenscience** [comment]
Yes, I have seen this repo and also this one too. Inspired from this, I wanted to give the exact steps, one by one! 👍

**Comment by aritra120** [comment]
Yes Sir I have seen that. Actually, according to me that guideline may become a bit longer, that is why I have sent you that one. Otherwise, that one is super clear and easy to learn things for beginners.

**Comment by praveenscience** [comment]
@aritra120 Sir, you talk that and that, which one are you talking about? Hehe.

**Comment by Deek9399** [comment]
@praveenscience We can stick to Contributing to Rezume, things are clearly explained even for beginners. I am an Ubuntu user, if there is a need for guidelines for ubuntu. I would like to work on it.

**Comment by praveenscience** [comment]
@Deek9399 So you know what needs to be done right? Would you like to pick it up then?

**Comment by Deek9399** [comment]
Yes sure. 
1. Create a contribute.md with a git flow particular to this project  ie. has screenshots taken on ubuntu while working with this repo.
2. Some contribution best practices. I would like to add similar to one in https://github.com/fossasia/open-event-frontend. If you verify and approve.
Let me know if I am on a right track or need some corrections.

**Comment by Deek9399** [comment]
Okay... I will start doing it.

**Comment by praveenscience** [comment]
@Deek9399 One of the mentors, @tejan-singh actually made it slightly easier for you by splitting up the files, so you can work on top of it okay? 😊

**Comment by Deek9399** [comment]
Okay.... will get in touch with him for any confusion/suggestion.

**Comment by praveenscience** [comment]
@Deek9399 Everything's done, just check it out once, replace the contents of the steps with how it is in Rezume. Then you're good to go. 😊

**Comment by Deek9399** [comment]
Yes, I checked it. Everything is nicely done. I will try to finish the steps section by tomorrow evening.

**Comment by praveenscience** [comment]
Yay!!! Thanks a lot @Deek9399, waiting for your PR.

**Comment by Deek9399** [comment]
Just made a PR. Please review and let me know if any further improvements are required.
